casper/networks.py

```diff
@@ -0,0 +1,37 @@
+import random as r
+
+from casper.network import Network
+
+
+class SynchronousNetwork(Network):
+    def delay(self, sender, receiver):
+        return 0
+
+
+class StepNetwork(Network):
+    def delay(self, sender, receiver):
+        return 1
+
+
+class ConstantDelayNetwork(Network):
+    CONSTANT = 5
+
+    def delay(self, sender, receiver):
+        return self.CONSTANT
+
+
+class LinearDelayNetwork(Network):
+    MAX_DELAY = 5
+
+    def delay(self, sender, receiver):
+        return r.randint(1, self.MAX_DELAY)
+
+
+class GaussianDelayNetwork(Network):
```

Comment:
Cool. What's great about having these as classes is we can store a lot more internal state! Good stuff :D

Reply:
Yes like adding mu and sigma to the state a' la' https://gist.github.com/SeanAvery/1832b257c7fc7a5d0428ad03914e0c27 :)